cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaBrokerConfigurationBuilder.java

```diff
@@ -239,6 +255,7 @@ public KafkaBrokerConfigurationBuilder withKRaft(String clusterName, String name
      *
      * @return Returns the builder instance
      */
+    @SuppressWarnings({"checkstyle:CyclomaticComplexity"})
```

Comment:
If you see the actual changes in the code, there are just some `if` conditions changed and a couple of addition. I don't think it's more complex than before. I guess I needed this addition because we are living border-line already on the current main. Tbh taking into account this code could be simplify when ZooKeeper will be totally dropped with KRaft only support, I would not bother that much to break it now.

Reply:
Well, I can live with it if needed. But:
* Zoo will be dropped in ~ 1 year
* Clearly, the `CyclomaticComplexity` increased. So it is hard to arque that you did not increased the complexity 😉

Reply:
I am not saying I didn't increase that, but maybe we are were border line ... just one `if` made me the bad guy :-D
Anyway, let's leave this refactoring for a different PR later I would say.